packages/react/src/Image/Image.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, ImgHTMLAttributes } from 'react'
+
+export interface ImageProps extends ImgHTMLAttributes<HTMLImageElement> {}
+
+export const Image = forwardRef(({ className, ...restProps }: ImageProps, ref: ForwardedRef<HTMLImageElement>) => (
+  <img {...restProps} ref={ref} className={clsx('amsterdam-image', className)} />
```

Comment:
Are we going to have additional components for figure/figcaption or responsive images?

Reply:
Yes, I can see a approach like `<Figure caption="…"><Image /></Figure>` in a later change. This is just the MVP Image component.

As `Image` extends the HTML attributes of `img`, implementers can provide multiple resolutions of the image using the `srcset` and `sizes` attributes to let the browser download the appropriate source.

If various the sizes need different cutouts of the image (the ‘[art direction](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction)’ aspect), we may offer a React component to wrap the `picture` element later.